authorize_format

```diff
@@ -0,0 +1,6 @@
+# Save as authorize.py
+import spotipy
```

Comment:
Move this to settings, and then pick the value from env variables.

You may use `.env`

Reply:
Wow. Just learning how a thing can be done in so many different ways!
And this way is more easier given that pipenv loads .env automatically :)
Fixed in: https://github.com/fossdelhi/sync-music/commit/380bd458ec7a1f2c3a15d461f5cc9de32109085c